tkdesigner/figma/frame.py

```diff
@@ -34,17 +34,17 @@ def __init__(self, node, figma_file, output_path):
 
     def create_element(self, element):
         element_name = element["name"].strip()
-        element_type = element["type"].strip()
+        element_type = element["type"].strip().lower()
 
         print(
             "Creating Element "
             f"{{ name: {element_name}, type: {element_type} }}"
         )
 
-        if element_name == "Rectangle":
+        if element_type == "rectangle":
             return Rectangle(element, self)
 
-        elif element_name == "Button":
+        elif element_type == "button":
```

Comment:
Button does not work on Element type. You need to manually name it as button in your Figma design to get the Rectangle to work as button.

The objects and shapes are converted based on Layer names and not the types. So please revert the changes you made to this section of code.

Reply:
Only the ones where you changed element_name to element_type. And leave the names lowercase.